src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.SpecialTypes.cs

```diff
@@ -148,4 +148,40 @@ static void TestAction([AsParameters] ParametersListWithHttpContext args)
         Assert.Same(httpContext.Request, httpContext.Items["request"]);
         Assert.Same(httpContext.Response, httpContext.Items["response"]);
     }
+
+    [Fact]
+    public async Task RequestDelegatePopulatesParametersWithDefaultValues()
```

Comment:
Do we have any other coverage for default values?
If not, this feels like a good opportunity to add a few more. Like `bool value = true`, `string s = "test"`

Reply:
There's a few tests sprinkled in other places but it's a good idea to put them in one place.